tests/components/minecraft_server/test_config_flow.py

```diff
@@ -0,0 +1,177 @@
+"""Test the Minecraft Server config flow."""
+
+from unittest.mock import PropertyMock, patch
```

Comment:
Import `patch` from `asynctest` instead. It's backwards compatible and can patch both coroutines and normal functions automatically.